lib/macho/macho_file.rb

```diff
@@ -302,7 +302,7 @@ def dylib_id
 
       dylib_id_cmd = command(:LC_ID_DYLIB).first
 
-      dylib_id_cmd.name.to_s
```

Comment:
Let's preserve the original API uses here -- `path`, `name`, etc. are more semantic and not context-dependent the way `to_s` is.